science/nco/Portfile

```diff
@@ -4,10 +4,11 @@ PortSystem          1.0
 PortGroup           compilers 1.0
 PortGroup           github 1.0
 
-github.setup        nco nco 5.0.3
-revision            3
+github.setup        nco nco 5.0.4
+revision            0
 platforms           darwin
-maintainers         {takeshi @tenomoto}
+maintainers         {takeshi @tenomoto} \
+                    @remkos
```

Comment:
I would say lets make this `openmaintainer` as well:
```
maintainers         {takeshi @tenomoto} \
                    @remkos \
                    openmaintainer
```

Reply:
And also, it would be nice to get an email address for the new maintainer (in {}).

Reply:
Thanks for the suggestions, @herbygillot and @pmetzger 
Done as requested and squashed commits.